src/pages/Main/EmailSection.js

```diff
@@ -24,8 +24,8 @@ export default function EmailSection() {
   const handleSubmit = async e => {
     e.preventDefault();
     try {
-      const result = await Ajax.post(formUrl, getFormData());
-      setSubscribed(result.inlineMessage);
+      await Ajax.post(formUrl, getFormData());
+      setSubscribed(true);
     } catch (err) {
       // Show error message setError(true); "Please submit a valid e-mail address."
```

Comment:
Can we show a validation error if the email is not valid? I think we use Formik in other forms to handle this.